build/media_source/com_scheduler/js/admin-view-config-helper.es6.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+if (!window.Joomla) {
+  throw new Error('Joomla API was not properly initialised!');
+}
+
+const ruleTypesElement = document.querySelector('#jform_execution_rules_rule_type');
+
+const elements = [
+  '#jform_execution_rules_cron_expression_minutes',
+  '#jform_execution_rules_cron_expression_hours',
+  '#jform_execution_rules_cron_expression_days_month',
+  '#jform_execution_rules_cron_expression_months',
+  '#jform_execution_rules_cron_expression_days_week'
+];
+
+function toggleRequired(process = 'add') {
+  if (!['add', 'remove'].contains(process)) {
```

Comment:
```suggestion
  if (!['add', 'remove'].includes(process)) {
```
Add suggestion by @n3t .